runtime/jcl/common/thread.cpp

```diff
@@ -326,18 +326,56 @@ Java_java_lang_Thread_getStackTraceImpl(JNIEnv *env, jobject rcv)
 	J9VMThread *currentThread = (J9VMThread*)env;
 	J9JavaVM *vm = currentThread->javaVM;
 	J9InternalVMFunctions *vmFuncs = vm->internalVMFunctions;
+	jobject result = NULL;
```

Comment:
if the thread is a vthread, and the inspector count is negative will this return NULL? does the API allow this?

Reply:
This allowed only for VirtualThread, the caller API runs in a loop until non-null value (or exception) is returned. see https://github.com/ibmruntimes/openj9-openjdk-jdk19/blob/openj9/src/java.base/share/classes/java/lang/VirtualThread.java#L854-L864